### PR TITLE
Windows Phone 8 reports user-agent with 'Windows Phone 8.0', so modified...

### PR DESCRIPTION
**Pull request by rawkode**
... pattern match to utilise 'windows phone' rather than 'windows phone os'

### CODE REVIEW

**Comment by lsmith77** [comment]
looks like something unrelated to your change is broken in our CI setup. do you by chance have time to have a look and try and fix it?

**Comment by rawkode** [comment]
Yeah, @lsmith77. I'll take a look

**Comment by rawkode** [comment]
The problem is that sInce Symfony 2.4, symfony/expression-language exists and is an external dependency

**Comment by lsmith77** [comment]
maybe something like this would help? https://github.com/FriendsOfSymfony/FOSRestBundle/blob/master/.travis.yml#L17

**Comment by rawkode** [comment]
Hi @lsmith77,

```ExpressionFunctionProviderInterface``` is provided by ```symfony/security-bundle:2.4.*``` - so I've added it to ```composer.json```, rather than adding/depending on ```symfony/expression-language:2.6.*```

Or would you rather that ```symfony/expression-language:2.6.*``` was the dependency?

Thanks,
David

File: composer.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
     ],
     "require": {
         "php": ">=5.3.3",
-        "symfony/framework-bundle": "~2.0"
+        "symfony/framework-bundle": "~2.0",
+        "symfony/expression-language": "~2.6"
```
Comment (lsmith77) [inline composer.json]: can't we move that to require-dev?

**Comment by lsmith77** [comment]
I have created https://github.com/liip/LiipThemeBundle/pull/111

**Comment by rawkode** [comment]
I guess it could be require-dev. I'll watch #112 for updates. thanks for your assistance, @lsmith77

**Comment by lsmith77** [comment]
I merged #112 .. thanks!

**Pull request closed without merging**